### PR TITLE
Inconsistency in shape of cell state in ConvLSTM cell rectified

### PR DESCRIPTION
**Pull request by ghost**
Fixes issue #21400 : next_state in LSTMStateTuple of ConvLSTM cell should have shape (state_size, output_size) but found (state_size,state_size)

### CODE REVIEW

**Comment by alphastreak** [comment]
I signed it!

File: tensorflow/contrib/rnn/python/ops/rnn_cell.py [hunk 1]
```diff
@@ -2097,7 +2097,7 @@ def __init__(self,
 
     state_size = tensor_shape.TensorShape(
         self._input_shape[:-1] + [self._output_channels])
-    self._state_size = rnn_cell_impl.LSTMStateTuple(state_size, state_size)
+    self._state_size = rnn_cell_impl.LSTMStateTuple(state_size, self._output_size)
     self._output_size = tensor_shape.TensorShape(
```
Comment (ebrevdo) [inline tensorflow/contrib/rnn/python/ops/rnn_cell.py]: Isn't output size defined after this line?

**Review by ebrevdo — CHANGES_REQUESTED** [review]

File: tensorflow/contrib/rnn/python/ops/rnn_cell.py (hunk 1 above)
Comment (alphastreak) [reply tensorflow/contrib/rnn/python/ops/rnn_cell.py]: Oh sorry. I will correct that now

Comment (alphastreak) [reply tensorflow/contrib/rnn/python/ops/rnn_cell.py]: Please review the updated patch

**Review by ebrevdo — APPROVED** [review]

**Comment by alphastreak** [comment]
This is my first contribution. So I am curious to know how much time would it take to run the above checks.

**Comment by qlzh727** [comment]
The lint is failing for this PR. Please fix the code accordingly. 

FAIL: Found 1 non-whitelited pylint errors:
tensorflow/contrib/rnn/python/ops/rnn_cell.py:2102: [C0301(line-too-long), ] Line too long (82/80)

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by tensorflowbutler** [comment]
It has been 43 days that this pull-request has stalled. Please create a new pull-request with the requested changes.

**Pull request closed without merging**